pkg/cmd/enable/addons/exec.go

```diff
@@ -117,15 +118,15 @@ func (o *Options) runWithClient(clusterClient clusterclientset.Interface,
 	for _, addon := range o.values.addons {
 		if addon == appMgrAddonName {
 			for _, clusterName := range o.values.clusters {
-				cn := &ClusterName{ClusterName: clusterName}
+				cn := &ClusterName{ClusterName: clusterName, NameSpace: o.namespace}
```

Comment:
I think we should give the name space a default value `open-cluster-management-agent-addon`